rollup.config.js

```diff
@@ -16,11 +16,10 @@ export default [
   {
     input,
     context: 'window',
-    output: {
-      name,
-      file: pkg.browser,
-      format: 'iife'
-    },
+    output: [
+      { name, file: 'dist/snapshot.min.js', format: 'iife' },
+      { name, file: pkg.browser, format: 'amd' }
```

Comment:
These examples suggest we should use UMD
https://github.com/rollup/rollup-starter-lib/blob/master/rollup.config.js
https://medium.com/@adostes/authoring-a-javascript-library-that-works-everywhere-using-rollup-f1b4b527b2a9
Can you try if it work? Also we can remove the IIEF build and use min.js for the UMD version

Reply:
Replaced iife and amd with umd 👍